### PR TITLE
[CPU] [ARM] Enable Conv SLT tests on ARM

### PR DESCRIPTION
**Pull request by alvoron**

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp [hunk 1]
```diff
@@ -469,7 +469,10 @@ std::vector<CPUSpecificParams> filterCPUInfoForDevice(const std::vector<CPUSpeci
             continue;
         if (selectedTypeStr.find("amx") != std::string::npos && !InferenceEngine::with_cpu_x86_avx512_core_amx())
             continue;
-
+#if !defined(OPENVINO_ARCH_ARM) && !defined(OPENVINO_ARCH_ARM64)
```
[the remark this answers is not in the document]
Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp]: This method was refactored in another PR. Define was removed.

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/convolution.cpp [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+            //dynamic shape
+            { {1, 200}, 12, {1, 200} },
+            { //target static shapes
+                { 2, 12, 7 },
+                { 1, 12, 5 }
+            }
+        }
+};
+
+const std::vector<fusingSpecificParams> fusingParamsSetWithEmpty{
```
[the remark this answers is not in the document]
Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/convolution.cpp:45]: Duplicated objects has been moved to  common section

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 3]
```diff
@@ -217,6 +217,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(smoke_NegativeQuantizedMatMulMultiplyFusion.*)");
     // int8 specific
     retVector.emplace_back(R"(smoke_Quantized.*)");
+    //TODO: oneDNN does not support scale shift fusing on ARM
+    retVector.emplace_back(R"(smoke_Conv_2D_FP32_dynBatch.*Fused=Relu.Multiply\(PerChannel\)\.Add\(PerChannel\))");
```
[the remark this answers is not in the document]
Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp]: Skip rules were removed.

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 4]
```diff
@@ -217,6 +217,12 @@ std::vector<std::string> disabledTestPatterns() {
...
+    //TODO: oneDNN does not support NCHW convolution
+    retVector.emplace_back(R"(.*Conv_2D_.*inFmts=nchw_outFmts=nchw_primitive=gemm_acl.*)");
+    //TODO: oneDNN does not support 3D convolution on ARM
+    retVector.emplace_back(R"(.*Conv_3D.*)");
```
[the remark this answers is not in the document]
Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp]: Skip rules were removed.

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp [hunk 5]
```diff
@@ -217,6 +217,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(smoke_NegativeQuantizedMatMulMultiplyFusion.*)");
     // int8 specific
     retVector.emplace_back(R"(smoke_Quantized.*)");
+    //TODO: oneDNN does not support scale shift fusing on ARM
+    retVector.emplace_back(R"(smoke_Conv_2D_FP32_dynBatch.*Fused=Relu.Multiply\(PerChannel\)\.Add\(PerChannel\))");
+    //TODO: oneDNN does not support NCHW convolution
+    retVector.emplace_back(R"(.*Conv_2D_.*inFmts=nchw_outFmts=nchw_primitive=gemm_acl.*)");
```
[the remark this answers is not in the document]
Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp]: Skip rules were removed.

**Review by EgorDuplensky — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/convolution.cpp [hunk 6]
```diff
@@ -0,0 +1,546 @@
...
+                                         ::testing::Values(ElementType::undefined),
+                                         ::testing::Values(ElementType::undefined),
+                                         ::testing::ValuesIn(inShapesGemm2D_cache()),
+                                         ::testing::Values(ov::test::utils::DEVICE_CPU)),
+                                 ::testing::ValuesIn(filterCPUInfo(CPUParams_GEMM_2D())),
+                                 ::testing::Values(emptyFusingSpec),
+                                 ::testing::Values(cpuEmptyPluginConfig)),
+                         ConvolutionLayerCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(smoke_Conv_2D_GEMM_I8, ConvolutionLayerCPUTest,
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common/convolution.cpp:69]: https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/ie/job/ie-tests-linux-debian_9_arm-cpu/2127/#showFailuresLink

**Pull request merged**